homeassistant/components/sensor/nut.py

```diff
@@ -217,7 +217,9 @@ def opp_state(self):
             return STATE_TYPES['OFF']
         else:
             try:
-                return STATE_TYPES[self._data.status[KEY_STATUS]]
+                return " ".join(
+                    STATE_TYPES[state]
```

Comment:
If one of the types is not found, the whole state will be STATE_UNKNOWN.

I'm fine with it but just wanted to point it out.